### PR TITLE
Action workflow to deploy docs website

### PR DESCRIPTION
**Pull request by mkam039**

### CODE REVIEW

**Comment by chamini2** [comment]
Why the change from yarn to npm? It seems the docusaurus downgrade _should_ be enough?

**Comment by mkam039** [comment]
> Why the change from yarn to npm? It seems the docusaurus downgrade _should_ be enough?

@chamini2, npm did a much better job resolving vulnerabilities

**Review by chamini2 — APPROVED** [review]

File: .github/workflows/docs_deployment.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Deploy docs website
+
+on:
+  push:
+    branches: [main, release]
```
Comment (chamini2) [inline .github/workflows/docs_deployment.yml:5]: Interesting, maybe should be only release? As it Is the released code?

Comment (mkam039) [reply .github/workflows/docs_deployment.yml:5]: We're doing two different deployments according to the branch that's being pushed to. I think it's good to test doc deployment with the main branch before we do prod deployment.

Comment (chamini2) [reply .github/workflows/docs_deployment.yml:5]: You are right! Great!

**Pull request merged**